### PR TITLE
Remove hardcoded code for root endpoint.

### LINKED ISSUE #435: Remove hardcoded supported chain IDs from '/'
We should remove the hardcoded supported chains from '/' https://github.com/oceanprotocol/ocean-node/blob/develop/src/components/httpRoutes/rootEndpoint.ts. Check status command for it or just show the endpoints.

### PR DESCRIPTION
**Pull request by mariacarmina**
Fixes #435  .

Changes proposed in this PR:

- complete the endpoints for `/`
- fix chain ids list
- tested on remote VM

### CODE REVIEW

**Comment by mariacarmina** [comment]
Response from root endpoint now:

Screenshot 2024-05-21 at 15 41 03

File: src/OceanNode.ts [hunk 1]
```diff
@@ -23,7 +23,8 @@ export class OceanNode {
     private db?: Database,
     private node?: OceanP2P,
     private provider?: OceanProvider,
-    private indexer?: OceanIndexer
+    private indexer?: OceanIndexer,
+    private config?: OceanNodeConfig
```
Comment (paulo-ocean) [inline src/OceanNode.ts:27]: we don't need config here at all, we already removed it in the past. Lets not do it again
tkx

Comment (paulo-ocean) [reply src/OceanNode.ts:27]: this is enough:
```
rootEndpointRoutes.get('/', async (req, res) => {
  const config = await getConfiguration()
```
no need to complicate

Comment (mariacarmina) [reply src/OceanNode.ts:27]: Ok, I'll make it async then, thanks

**Comment by jamiehewitt15** [comment]
This is good, but it would be even better if the route endpoints are also dynamically generated, rather than hardcoded in a json. As you've seen that list tends to be ignored and it will probably become out of date again soon. It would be nice if we don't have to maintain it.

**Comment by paulo-ocean** [comment]
> This is good, but it would be even better if the route endpoints are also dynamically generated, rather than hardcoded in a json. As you've seen that list tends to be ignored and it will probably become out of date again soon. It would be nice if we don't have to maintain it.

I agree, this will get outdated again probably, but probably better do it on another PR

**Review by paulo-ocean — APPROVED** [review]
lgtm

**Review by jamiehewitt15 — APPROVED** [review]
Looks good to me, the endpoints will be handled in another PR

**Pull request merged**